design/workitemtype.go

```diff
@@ -21,9 +21,15 @@ var fieldDefinition = a.Type("fieldDefinition", func() {
 	a.Description("A fieldDescription aggregates a fieldType and additional field metadata")
 	a.Attribute("required", d.Boolean)
 	a.Attribute("type", fieldType)
-
-	a.Required("required")
-	a.Required("type")
+	a.Attribute("label", d.String, "A label for the field that is shown in the UI", func() {
+		a.Example("Iteration")
+		a.MinLength(1)
+	})
+	a.Attribute("description", d.String, "A description for the field", func() {
+		a.Example("The iteration field tells to which iteration a work item belongs.")
```

Comment:
ah! its value `iteration field tells...` for the attribute `description`  was confusing